service/grpc/codelingo/codelingo.pb.go

```diff
@@ -276,27 +171,6 @@ func (m *ListFactsRequest) String() string            { return proto.CompactText
 func (*ListFactsRequest) ProtoMessage()               {}
```

Comment:
exported method ListFactsRequest.Descriptor should have comment or be unexported